src/libraries/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -4,6 +4,8 @@
 // Changes to this file must follow the https://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System.Runtime.Serialization;
+
```

Comment:
```suggestion
```
looks like tool didn't removed unused usings.

Reply:
No, my mistake. I'll fix

Reply:
@am11, yeah, you're not the first to mention it, @stephentoub found this too, but if I remove it, I cannot build anymore and I assume the CI would fail too as there's a dependency. What's your suggestion?

Reply:
The tool fully qualifies everything.

Reply:
@abelbraaksma, Dan has already fixed it by running the tool. I was commenting on post-tool-run issue that it hasn't removed the unused using after inlining the fully qualified names. ;)

Reply:
The tool made some unrelated edits, in removing those I had put the using back...

Reply:
(btw: I didn't touch this ;) )

Reply:
learning something new every day (but then again: I didn't use the tool, will do so next time!)